src/community/json-ld/src/main/java/org/geoserver/jsonld/configuration/JsonLdTemplateReader.java

```diff
@@ -121,13 +136,47 @@ private void getBuilderFromJsonArray(
 
     private void getBuilderFromJsonAttribute(
             String nodeName, JsonNode node, JsonBuilder currentBuilder) {
+        String strNode = node.asText();
+        String filter = null;
+        if (strNode.contains(FILTERKEY)) {
+            String[] arrNode = strNode.split(",");
```

Comment:
What if the filter itself contains a comma? (a CQL filter can contain a "in" operator that uses it, or might have a string literal containing it).

Reply:
fixed